src/python/grpcio/_unixccompiler_patch.py

```diff
@@ -32,90 +32,43 @@
 from distutils import ccompiler
 from distutils import errors
 from distutils import unixccompiler
+from distutils import util
 import os
 import os.path
 import shutil
 import sys
 import tempfile
 
+def _unix_commandfile_spawn(unused_self, command, *args, **kwargs):
```

Comment:
Erp, the `*CCompiler` classes are old-style, so `super()` doesn't work with them. Just going to directly call the `CCompiler.spawn` method.